apps/mds_docs/src/app/globals.css

```
@@ -0,0 +1,92 @@
+  --radius-chip: 100px;
+  /* Typography tokens */
+  --typography-font-family-brand: Pretendard;
+  --typography-font-size-caption-tiny: 10px;
+  --typography-font-size-chip-label: 13px;
+  --typography-font-size-app-bar-title: 18px;
+  --typography-font-weight-medium: 500;
+  --typography-font-weight-semi-bold: 600;
+}
+
+@font-face {
+  font-family: 'Pretendard';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)